website/docs/r/event_orchestration_unrouted.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_event_orchestration_unrouted"
+sidebar_current: "docs-pagerduty-resource-event-orchestration-unrouted"
+description: |-
+  Creates and manages an Unrouted Orchestration for a Global Event Orchestration in PagerDuty.
+---
+
+# pagerduty_event_orchestration_unrouted
+
+An Unrouted Orchestration allows users to create a set of Event Rules that will be evaluated against all events that don't match any rules in the Orchestration's Router.
+
+The Unrouted Orchestration evaluates events sent to it against each of its rules, beginning with the rules in the "start" set. When a matching rule is found, it can modify and enhance the event and can route the event to another set of rules within this Unrouted Orchestration for further processing.
+
+## Example of configuring a Unrouted Rules for an Orchestration
+
+  In this example the user has defined the unrouted orchestration with two rules, each routing to a different service.This example assumes services used in the route_to configuration already exists. So it does not show creation of service resource.As there is a single set, route_to is not defined for the rules.
+
+  If there are more than one set, the rules in start set must define route_to with id of the next set
+  The first rule matches only if the condition is matched. The second rule matches always as there are no conditions.
+
+  `catch_all` with empty `actions` block suppresses the alerts that do not match any rules. Using the `catch_all` to set severity, event_action, variables and extractions is also allowed.
+
+```hcl
+resource "pagerduty_event_orchestration_unrouted" "unrouted" {
+ event_orchestration = pagerduty_event_orchestration.my_monitor.id
+  sets {
+    id = "start"
+    rules {
+      label = "Update the summary of un-matched Critical alerts so they're easier to spot"
+      conditions {
+        expression = "event.severity matches 'critical'"
+      }
+      actions {
+        severity = "info"
+        extractions {
+            target = "event.summary"
+            template = "[Critical Unrouted] {{event.summary}}"
+        }
+      }
+    }
+    rules {
+      label = "Reduce the severity of all other unrouted events"
+      actions {
+        severity = "info"
+      }
+    }
+  }
+  catch_all {
+    actions { }
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `event_orchestration` - (Required) The Event Orchestration to which this Unrouted Orchestration belongs to.
+* `sets` - (Required) An Unrouted Orchestration must contain at least a "start" set, but can contain any number of additional sets that are routed to by other rules to form a directional graph.
+* `catch_all` - (Required) When none of the rules match an event, the event will be routed according to the catch_all settings.
```

Comment:
**Blocker:** Same as above, in an Unrouted Orchestration the `catch_all` cannot do routing. Maybe this instead?

```suggestion
* `catch_all` - (Required) the `catch_all` actions will be applied if an Event reaches the end of any set without matching any rules in that set.
```